keiko-sql/src/main/kotlin/com/netflix/spinnaker/config/SqlQueueProperties.kt

```diff
@@ -0,0 +1,49 @@
+package com.netflix.spinnaker.config
+
+import com.netflix.spinnaker.kork.sql.config.SqlRetryProperties
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.validation.annotation.Validated
+import javax.validation.constraints.Pattern
+import javax.validation.constraints.Positive
+
+@ConfigurationProperties("keiko.queue.sql")
+@Validated
+class SqlQueueProperties {
+  /**
+   * Enables use of the SqlQueue implementation, disabling RedisQueue.
+   */
+  var enabled: Boolean = false
+  /**
+   * [queueName] namespaces the database tables so that multiple keiko queues can be collocated
+   * on the same database. When using a globally writeable data store with queue processors in
+   * multiple regions, it may be desirable to use the region as the [queueName]. Must match
+   * regexp "\w+".
+   */
+  @Pattern(regexp = "^\\w+$")
+  var queueName: String = "default"
+  /**
+   * [deadLetterQueueName] defaults to [queueName] but can be set independently so that multiple
+   * queues on the same database can potentially share a DLQ. Must match regexp "\w+".
+   */
+  @Pattern(regexp = "^\\w+$")
+  var deadLetterQueueName: String = queueName // Can be shared across queues on the same master
```

Comment:
Strong desire to avoid inline comments like here. Can we promote this comment into the property docs?